testinfra/app-code/test_securedrop_app_code.py

```diff
@@ -35,6 +35,17 @@ def test_securedrop_application_apt_dependencies(Package, package):
     assert Package(package).is_installed
 
 
+def test_securedrop_application_test_locale(File, Sudo):
+    """
+    Ensure SecureDrop DEFAULT_LOCALE is present.
+    """
+    securedrop_config = File("{}/config.py".format(
+        securedrop_test_vars.securedrop_code))
+    with Sudo():
+        assert securedrop_config.is_file
+        assert securedrop_config.contains("DEFAULT_LOCALE")
```

Comment:
Nit: these tests could be a bit stricter, testing for two things:

* that the DEFAULT_LOCALE config line is formatted correctly (e.g. anchor it with `"^DEFAULT_LOCALE"`)
* that the DEFAULT_LOCALE config line occurs _only once_, since we have concerns about idempotence here

Will test locally and report back.

Reply:
Yup, like this:

```
diff --git a/testinfra/app-code/test_securedrop_app_code.py b/testinfra/app-code/test_securedrop_app_code.py
index 76e4e955..6dbef49a 100644
--- a/testinfra/app-code/test_securedrop_app_code.py
+++ b/testinfra/app-code/test_securedrop_app_code.py
@@ -43,7 +43,8 @@ def test_securedrop_application_test_locale(File, Sudo):
         securedrop_test_vars.securedrop_code))
     with Sudo():
         assert securedrop_config.is_file
-        assert securedrop_config.contains("DEFAULT_LOCALE")
+        assert securedrop_config.contains("^DEFAULT_LOCALE")
+        assert securedrop_config.content.count("DEFAULT_LOCALE") == 1
 
 
 def test_securedrop_application_test_journalist_key(File, Sudo):
```

With that change, satisfied for merge!